lib/Conversion/TorchOnnxToTorch/DefaultDomainGtoP.cpp

```diff
@@ -1894,6 +1894,36 @@ void mlir::torch::onnx_c::populateDefaultDomainGtoP(
         rewriter.replaceOpWithNewOp<Torch::AtenOneHotOp>(binder.op, resultType,
                                                          argmax, oneInt);
 
+        return success();
+      });
+  patterns.onOp(
+      "LpNormalization", 22,
```

Comment:
Can you reduce the opset version to 1? I don't think there is a significant diff between these versions from [the docs](https://onnx.ai/onnx/operators/onnx__LpNormalization.html)